src/protocol/meson.build

```diff
@@ -0,0 +1,19 @@
+if wayland_scanner.found()
+	prog_wayland_scanner = find_program(wayland_scanner.get_pkgconfig_variable('wayland_scanner'))
+else
+	prog_wayland_scanner = find_program('wayland-scanner')
+endif
+
+gen_client_header = generator(prog_wayland_scanner,
+	output: ['@BASENAME@-client.h'],
+	arguments: ['-c', 'client-header', '@INPUT@', '@BUILD_DIR@/@BASENAME@-client.h'])
+
+gen_private_code = generator(prog_wayland_scanner,
+	output: ['@BASENAME@.c'],
+	arguments: ['-c', 'private-code', '@INPUT@', '@BUILD_DIR@/@BASENAME@.c'])
```

Comment:
Can you replace with `code` and add in the comment about it being deprecated. I think there are still platforms with the old libwayland and using the deprecated `code` is harmless.

Reply:
@wmww Just my 5 cents, most such outdated platforms won't support `wlroots` (since it requires really new wayland-protocols) and probably the same for Mir.

Reply:
Unrelated. This project is not associated with wlroots.

Reply:
I know, but since wlroots and Mir are the major implementations of layer-shell, I thought it is relevant. Feel free to ignore me :)